docs/modules/ROOT/pages/configure-diagnostic-logging.adoc

```diff
@@ -0,0 +1,62 @@
+* File-based rolling limits can also be controlled via DiagnosticsFileConfiguration (maxRolledFileSizeInMB, maxRolledFileCount, fileNamePrefix) where applicable. See: https://docs.hazelcast.com/operator/latest-snapshot/api-ref#diagnosticsfileconfiguration[DiagnosticsFileConfiguration].
+
+// question - is the following in the right place or should it appear later under the system properties section? Please advise
+
+The diagnostics file rolls by default (10 files, 50 MB each) and contains plugin sections such as BuildInfo, SystemProperties, ConfigProperties, Metrics, SlowOperations, and HazelcastInstance. For information, see https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/monitoring#diagnostics-plugins[Diagnostics Plugins].
+
+*Example:*
+
+* outputType: choose the output destination (e.g. FILE or STDOUT). The default is STDOUT in Operator.
+* maxRolledFileSize / maxRolledFileCount: cap file size and retention.
+* fileNamePrefix: add a custom prefix.
+* pluginProperties: pass the same diagnostics system properties shown above (e.g. enable plugins or adjust their periods).
+* autoOffTimerInMinutes: configure automatic turn-off (see below).
+* pvcName: bind diagnostics to a PVC (see below).
+
+== Forward diagnostics to a logging framework
+
+Instead of file output, you can forward diagnostics to your logging framework by setting `spec.diagnostics.outputType=LOGGER`
+
+Then configure the com.hazelcast.diagnostics logger (e.g. a Log4j2 logger and rolling appender) to control size and rollover using time/size policies. For information, see https://docs.hazelcast.com/hazelcast/latest/maintain-cluster/monitoring#diagnostics-using-logging-frameworks[Diagnostics using Logging Frameworks].
+
+== Configure auto turn-off
+
+To prevent diagnostic logs from running indefinitely and having a performance impact on the cluster, set the Operator's `autoOffTimerInMinutes` to automatically disable diagnostic logging after the specified number of minutes. See https://docs.hazelcast.com/operator/latest-snapshot/api-ref#diagnostics[Diagnostics].
+
+You can combine this with rolling limits:
+
+* Operator diagnostics: maxRolledFileSize, maxRolledFileCount.
+* Hazelcast properties: hazelcast.diagnostics.max.rolled.file.size.mb and hazelcast.diagnostics.max.rolled.file.count.
+
+NOTE: When diagnostics logging is enabled with auto turn-off, members that join the cluster later will start their own timers from zero. As a result, earlier members may stop diagnostics before the last joiner, leading to a temporary inconsistency. To keep the reported state accurate, Operator restarts its diagnostics auto turn-off timer from zero whenever a new member joins while diagnostics is enabled. This ensures Operator shows diagnostics as enabled as long as at least one member is still logging. 
```

Comment:
The operator doesn't have a timer for auto turn-off, so we can remove this note entirely.

Reply:
Not all of them, just about the timer. Operator still supports the `auto-turn-off` feature provided by the core platform 😃